lib/htty/requests_util.rb

```diff
@@ -2,6 +2,7 @@
 require 'net/http'
 require 'net/https'
 require 'uri'
+require 'benchmark'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.